### PR TITLE
Nc news q5

### PR DESCRIPTION
**Pull request by rubyrubyruby02**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
 
-const {getArticle} = require('./controllers/article.controller')
-const {customErrorHandler} = require('./errors/errors')
+const {getTopics} = require('./controllers/topic.controller')
+const {getArticle, getAllArticles} = require('./controllers/article.controller')
 const {getEndpoints} = require('./controllers/endpoints.controller')
 
+const {customErrorHandler, PSQLErrorHandler} = require('./errors/errors')
+
+app.use(express.json())
```
Comment (dappernerddesigns) [inline app.js:10]: We're not doing any posting or patching, this isn't needed

File: models/articles.model.js [hunk 2]
```diff
@@ -13,17 +14,36 @@ const fetchArticle = (article_id)=> {
...
-        if(error.code === '22P02'){
-            return Promise.reject({
-                status: 400,
-                msg: "Bad request"
-            })
-        }
-        else {
-            return Promise.reject(error)
-        }
+        return Promise.reject(error)
```
Comment (dappernerddesigns) [inline models/articles.model.js:43]: This won't fire, psql will error before this if there is an issue. There shouldn't be any catch blocks in the model.

**Review by dappernerddesigns — APPROVED** [review]
Well done, this has meaty query. There's a few bits I'd like you to remove before you continue. Good job and merge away.

File: models/articles.model.js (hunk 2 above)
Comment (rubyrubyruby02) [reply models/articles.model.js:43]: Fab I've removed, retested and all working. Will now merge :) thank you

**Pull request merged**